lab_3/main.py

```diff
@@ -20,7 +21,42 @@ def tokenize_by_sentence(text: str) -> tuple:
          (('_', 'h', 'e', '_'), ('_', 'i', 's', '_'), ('_', 'h', 'a', 'p', 'p', 'y', '_'))
          )
     """
-    pass
+    if not isinstance(text, str):
+        return ()
+    text = text.lower()
+    for letter in text:
```

Comment:
Are you sure about checking *every* character? The number of special symbols is lesser. Consider iterating over them instead.

Reply:
Sorry, but in this case I don't understand how can I replace them